wfdb/io/recordinfo.py

```diff
@@ -0,0 +1,115 @@
+from dataclasses import dataclass
+import datetime
+from typing import List, Optional, Union
+
+
+@dataclass
+class SignalInfo:
+    """
+    Signal specification fields for one signal
+    """
+
+    file_name: Optional[str] = None
+    fmt: Optional[str] = None
+    samps_per_frame: Optional[int] = None
+    skew: Optional[int] = None
+    byte_offset: Optional[int] = None
+    adc_gain: Optional[float] = None
+    baseline: Optional[int] = None
+    units: Optional[str] = None
+    adc_res: Optional[int] = None
+    adc_zero: Optional[int] = None
+    init_value: Optional[int] = None
+    checksum: Optional[int] = None
+    block_size: Optional[int] = None
+    sig_name: Optional[str] = None
+
+
+class SignalSet:
```

Comment:
No unit tests for this because I need to think more about how these data classes are to be constructed.